packages/editor/src/components/copy-handler/index.js

```diff
@@ -75,13 +63,28 @@ export default compose( [
 		return {
 			hasMultiSelection: hasMultiSelection(),
 			selectedBlockClientIds,
+		};
+	} ),
+	withDispatch( ( dispatch, ownProps, { select } ) => {
+		const { getBlocksByClientId } = select( 'core/editor' );
+		const { removeBlocks } = dispatch( 'core/editor' );
+
+		return {
+			getSelectedBlocks: function() {
```

Comment:
`function` keyword seems non-conventional here, in place of the shorthand `getSelectedBlocks() {`